### DISCUSSION TITLE
Cannot use import statement outside a module

### DISCUSSION THREAD

**Discussion by prajaktapatilweb** [KevinVandy/material-react-table#163] [General]
I am using basic Material-react-table but it shows the error 
Cannot use import statement outside a module

and pointing to the file
node_modules/%20(mui/material/utils/debounce.js (1)

I am not able to understand how to remove this error.

 I am using Nextjs.

Please help

**Comment by KevinVandy** [comment]
Make sure you have up to date MUI V5 dependencies installed

**Comment by Booligoosh** [comment]
I get this error too when running jest on a JSX file importing material-react-table, even with the latest MUI v5 dependencies.

**Reply by KevinVandy** [reply]
what version of MRT do you have installed?

**Reply by KevinVandy** [reply]
There was an update to MRT 3 months ago that fixed this issue, so just make sure you have installed the latest version
